sensor.js

```diff
@@ -236,6 +242,22 @@ Rtl433Accessory.prototype = {
           }
 
           break;
+        case "contact":
+	  // {"time" : "2022-03-17 23:04:08.430623", "protocol" : 68, "model" : "Kerui-Security", "id" : 297536, "cmd" : 14, "opened" : 1, "state" : "open" }
+	  // {"time" : "2022-03-17 23:04:19.447761", "protocol" : 68, "model" : "Kerui-Security", "id" : 297536, "cmd" : 7, "opened" : 0, "state" : "close" }
+          // debug("update door this--->",this);
+          // console.log("update door this--->",this);
+
+          this.sensorService
+              .setCharacteristic(Characteristic.ContactSensorState, data.opened );
```

Comment:
Can you change this to

```
.setCharacteristic(Characteristic.ContactSensorState, ( data.opened === 1 ? Characteristic.ContactSensorState.CONTACT_DETECTED : Characteristic.ContactSensorState.CONTACT_NOT_DETECTED ));
```

Future proofs any changes on the Homebridge side

Reply:
yes